### DISCUSSION TITLE
Necromutant

### DISCUSSION THREAD

**Discussion by framerate** [secretcowlevel/DoomtrooperPublic#31] [Card Balance]
> This card may be played from your DISCARD pile as well as your HAND

Possibly part of a bigger change that also changes Necromutant Frenzy?

**Comment by 21-ric** [comment]
It's a big challenge. 

Let's try. 😎

NECROMUTANT (ALGEROTH) [C]
Gain +1 to [F], [S] and [A] for each UNDEADLEGIONNAIRE .
This card may be played from your DISCARD pile as well as your HAND
F: 4 S: 4 A: 4 V: 4

Centurion (ALGEROTH) [U]
Gain +2 to [F] [S] and [A] for each NECROMUTANT
F: 4 S: 4 A: 4 V: 5

Remove the card "Undead legionnaire"
You can only bring "undead legionnaire" by playing "portal of undeath" and "nepharite warlord".

Portal Of Undeath (ALGEROTH) [R]
FACTION, DARKSYMMETRY
Bestow only on a NEPHARITE. After this Warrior kills an enemy, an UNDEADLEGIONNAIRE joins your side.

Nepharite Warlord (ALGEROTH) [R]
NEPHARITE
At the begginning of your turn, create an Undead Legionnaires token into your Kohort. If you can't,
your Undead Legionnaires gain +1 to [F] and [A] .
F: 9 S: 6 A: 7 V: 8

Necromutant Frenzy (ALGEROTH) [U]
INFLUENCE
All Undead legionnaire in play or Necromutants if you don't have Undead legionnaire gain frontline.
Perhaps a little too strong. 😂

🤡

**Reply by framerate** [reply]
Good stuff, but I'm not sure we're there yet haha

One thing I did like though from a while back was getting rid of UNLIMITED keyword (since only one card has it) and making UL's playable from Discard (maybe instead of Necromutant?). So they're still pretty unlimited, they just keep coming back....

Not sure how to make it all work together though!

**Reply by Sol-au-fein** [reply]
Yes, but you have only 6 slots in your squad. How can you fit all of them in squad to make something useful?
Solution would be if centurion gain bonus for both legionnaires and necromutants.
I also feel someone should boost legionnaires in that build.

**Reply by Kite80** [reply]
what if DL has 6 extra warrior slots for legionnaires only? ALso we need this:

Necromakina (ALGEROTH) [R]
FACTION, SUPPORT

Every time you kill a warrior, put a counter on this card. You can put an Undead Legionnaire in play by spending 1 standard action; if you do so, remove a counter from the Necromakina.

**Comment by Sol-au-fein** [comment]
Necromutant: 
Living captives are brought to mechanized distortion chambers. Through the unholy process, the very structure of the victims muscles is changed, and the creature's basic genetic code reprogrammed. The result is soldier with nearly perfect combat abilities.
Necromutants still retain some activity in their brains and are capable of taking initiative of their own. They also retain part of their original personality. The small remains of human creativity in the Necromutants make them perfect commanders of small groups of legionnaires.

Centurion:
These creatures are the captains of Algeroth's kohorts, culled from cultists, captives and deserters from the humankind, brought to the distortion chambers od the Tekrons and utterly reshaped into beings of pure evil. Centurions are often placed in direct command over small groups of legionnaires and necromutants.
Centurions are fearsome humanoids standing over two meters tall, with bulging muscles and strong builds. Their leathery skin is grayish-green and very durable. They are often unarmored. They give out rank, musty odor, and their voices are loud, harsch and grating.

**Comment by Sol-au-fein** [comment]
To be honest i dont see problem with current Legionnaire - Necromutant - Centurion chain. 
Maybe necromutant should have some additional skill.

**Reply by 21-ric** [reply]
You right there is no problem. Just a lack of diversity.
To have a good deck you just have to play "undead legionnaire" and "centurion".
Necromutant is useless. 😶

**Reply by Sol-au-fein** [reply]
Lets say Centurion would gain +2/+2/+2 for both legionnaires and necromutants. Would that alone make necromutant more useful?
